src/Factory/Consignment/ConsignmentFactory.php

```diff
@@ -97,16 +100,31 @@ public function fromOrder(
         $this->setOrderData($order, $deliveryOptions);
         $this->createConsignment();
 
-        $collection        = new ConsignmentCollection();
-        $consignment       = $this->initConsignment();
-        $labelAmount       = $this->request['extraOptions']['labelAmount'] ?? 1;
+        $collection   = new ConsignmentCollection();
+        $extraOptions = new ExtraOptions();
+        $consignment  = $this->initConsignment();
+        $labelAmount  = $this->request['extraOptions']['labelAmount'] ?? 1;
+        $extraOptions->setLabelAmount($labelAmount);
         $distributedWeight = $consignment->getTotalWeight() / $labelAmount;
```

Comment:
Uitlijning = t.o.v. de =jes hierboven, of een witregel ertussen, zodat het er in ieder geval netjes uit ziet